README.rst

```diff
@@ -89,7 +89,7 @@ There are two date arithmetic traps in the Gregorian calendar:
 
 In both cases, the result must be changed to the first day of the next month.
 
-Provided periods are represented by [start date inclusive, end date exclusive),
+Provided periods are represented by (start date inclusive, end date exclusive),
```

Comment:
Upon further thought, it's likely going to confuse a large fraction of the intended audience, so I'll make the change anyway.